core/src/main/scala/org/apache/spark/h2o/backends/internal/SpreadRDDBuilder.scala

```diff
@@ -33,7 +34,7 @@ import scala.annotation.tailrec
   */
 private[spark]
 class SpreadRDDBuilder(@transient private val hc: H2OContext,
-                       numExecutorHint: Option[Int] = None) extends SharedBackendUtils {
+                       numExecutorHint: Option[Int] = None) extends Logging {
```

Comment:
Can you please help me understand why this has changed?

Reply:
Unnecessary import